.github/docker/linux/x64/Dockerfile

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2020 New Relic Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# ARGs passed from GHA workflow.
+#
+ARG PHP_VER
+ARG ARCH
+ARG BUILD_TYPE
+FROM php:${PHP_VER}
+
+RUN docker-php-source extract
+
+#
+#These args need to be repeated so we can propogate the VARS within this build context.
+#
+ARG PHP_VER
+ARG ARCH
+ARG BUILD_TYPE
+ENV PHP_VER=${PHP_VER}
+ENV ARCH=$ARCH
+ENV BUILD_TYPE=$BUILD_TYPE
```

Comment:
In the x86 version of this file, these are lower down in the file. That is advantageous because any change to these will result in all of the commands below them being rerun. We don't see the benefit until we start running these builds locally, but there's also a chance it will help GitHub reuse intermediate containers.

Either way, these should be in the same place across the two builds.

(We may also want to consider moving some of the ARGS from earlier in the file down as well (should they not be needed until later in the build)).